### ISSUE TITLE
Is this working on M1?

### ISSUE THREAD

**Issue by twhite96** [avdept/JellyBoxPlayer#28]
This is the screen I get when I open the app

- macOS: **14.4.1**
- Architecture: **arm64**
- latest JellyBoxPlayer from Mac App Store**
- Jellyfin Version: **10.8.13**
- Jellyfin Architecture: **x86**

![CleanShot 2024-04-05 at 20 41 14](https://github.com/avdept/JellyBoxPlayer/assets/7698292/09e85c84-84a8-44ed-a9c4-6e260368bd78)

**Comment by avdept** [comment]
Yes it works. Although it has few issues non related to platform but rather to Jellyfin server software itself. I received issue report in sentry so I'll have a look and fix it asap!

**Comment by avdept** [comment]
I submitted new version to appstore. It should fix the issue. Pls check in in few days and LMK if you still experience it
